tests/unit/hyperloglog.tcl

```diff
@@ -59,6 +59,35 @@ start_server {tags {"hll"}} {
         }
     } {} {needs:pfdebug}
 
+    test {Change hll-sparse-max-bytes} {
+        r config set hll-sparse-max-bytes 3000
+        r del hll
+        r pfadd hll a b c d e d g h i j k
+        assert {[r pfdebug encoding hll] eq {sparse}}
+        r config set hll-sparse-max-bytes 30
+        r pfadd hll new_element
+        assert {[r pfdebug encoding hll] eq {dense}}
+    } {} {needs:pfdebug}
+
+    test {Hyperloglog promote to dense well in different hll-sparse-max-bytes} {
+        set max 100
+        while {$max <= 3000} {
```

Comment:
3000 is because I hope to cover the default configuration of hll_sparse_max_bytes. It is prolonged because I test from 100 to 3000, with an increment of 100 each time. Now I changed it to only test 100, 500 and 3000, and now it uses about 500ms.